### ISSUE TITLE
New design in progress. 

### ISSUE THREAD

**Issue by teury** [ASKBOT/askbot-devel#890]
I have started working on a new design for askbot.
  
I plan to share it, although now I am starting. the askbot project is very big.

I have shown the best comment on the home page next to the question, like Quora.

I have also modified the home view and created a new class listview.

Jinja2 is discontinued and replaced by django templates.
#861
![Askbot2](https://user-images.githubusercontent.com/5114048/130305805-522e4f67-52fd-42eb-ac2f-7ffe8d93acb5.gif)

**Comment by evgenyfadeev** [comment]
Hi @teury, great. I recommend you stay in sync with this branch: https://github.com/ASKBOT/askbot-devel/tree/theme-cleanup I'm currently working on it, once it is complete it will be merged into the master branch and released with a new version.

I am making some changes to the class names and some ids that are necessary for the js to function properly.
Mainly I'm adding a js-... prefix to make clear that this class is important for the functionality.

Probably we cannot collaborate on the actual theme designs right now, but I hope that in the future there will be a way to to allow people use themes like the one you're working on.
